src/core/plugins/core-hud/client/index.ts

```diff
@@ -272,6 +274,20 @@ class InternalFunctions implements ViewModel {
     static defaultInteractionsComponent(propName) {
         InternalFunctions.passComponentInfo(propName, JSON.stringify(interactions.concat(customInteractions)), true);
     }
+
+
+    static defaultStreetComponent(propName: string) {
+        const pos = alt.Player.local.pos;
+        const streetHash = native.getStreetNameAtCoord(pos.x, pos.y, pos.z, null, null,)[1];
+        const value = native.getStreetNameFromHashKey(streetHash)
+        InternalFunctions.passComponentInfo(propName, value);
+    }
+    static defaultCrossStreetComponent(propName: string) {
```

Comment:
Let's keep consistency about the spaces between class methods.

```suggestion
    }

    static defaultStreetComponent(propName: string) {
        const pos = alt.Player.local.pos;
        const streetHash = native.getStreetNameAtCoord(pos.x, pos.y, pos.z, null, null,)[1];
        const value = native.getStreetNameFromHashKey(streetHash)
        InternalFunctions.passComponentInfo(propName, value);
    }
    
    static defaultCrossStreetComponent(propName: string) {
```